src/pudl/package_data/phmsagas/column_maps/yearly_transmission_gathering_summary_by_commodity.csv

```diff
@@ -24,36 +24,36 @@ system_state,stop,stop,stop,stop,stop,stop,stop,stop,stop,stop,stop,stop,stop,st
+hca_method_1_other_miles,,,,,,,,,,,,,,,,,,,,,,,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1,partb1othm1
+hca_method_1_total_miles,,,,,,,,,,,,,,,,,,,,,,,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t,partb1m1t
+hca_method_2_subject_matter_expert_miles,,,,,,,,,,,,,,,,,,,,,,,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2,partb1smem2
+hca_method_2_relative_risk_miles,,,,,,,,,,,,,,,,,,,,,,,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2,partb1rrm2
+hca_method_2_quantitative_miles,,,,,,,,,,,,,,,,,,,,,,,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2,partb1quanm2
+hca_method_2_probabilistic_miles,,,,,,,,,,,,,,,,,,,,,,,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2,partb1probm2
+hca_method_2_scenario_based_miles,,,,,,,,,,,,,,,,,,,,,,,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2,partb1sbm2
+hca_method_2_other_miles,,,,,,,,,,,,,,,,,,,,,,,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2,partb1othm2
+hca_method_2_total_miles,,,,,,,,,,,,,,,,,,,,,,,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t,partb1m2t
+hca_method_total_subject_matter_expert_miles,,,,,,,,,,,,,,,,,,,,,,,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet,partb1smet
+hca_method_total_relative_risk_miles,,,,,,,,,,,,,,,,,,,,,,,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt,partb1rrt
+hca_method_total_quantitative_miles,,,,,,,,,,,,,,,,,,,,,,,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant,partb1quant
+hca_method_total_probabilistic_miles,,,,,,,,,,,,,,,,,,,,,,,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt,partb1probt
+hca_method_total_scenario_based_miles,,,,,,,,,,,,,,,,,,,,,,,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt,partb1sbt
+hca_method_total_other_miles,,,,,,,,,,,,,,,,,,,,,,,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht,partb1otht
+hca_method_total_miles,,,,,,,,,,,,,,,,,,,,,,,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt,partb1mt
```

Comment:
this is not blocking but i think it will be helpful in our future transformations of these tables to include totals even when it seems silly/redundant. which i think in this structure would be `hca_method_total_total_miles`. but these guys might be easier to see and change when we actually get to transformations so i don't feel strongly at all about doing this right now because it would be relatively challenging and maybe preemptive to do it before we really know how its going to be reshaped.

Reply:
Agreed, but I think this will be easier to do during the transform step.